### ISSUE TITLE
when I use cfg_file cfgs/kitti_models/pv_rcnn.yaml ,in eval stage,Segmentation fault happens

### ISSUE THREAD

**Issue by dieforyou1** [open-mmlab/OpenLidarPerceptron#1758]
python train.py --cfg_file cfgs/kitti_models/pv_rcnn.yaml  --workers 1 --epochs 10 --batch_size 8

UserWarning: The torch.cuda.*DtypeTensor constructors are no longer recommended. It's best to use methods such as torch.tensor(data, dtype=*, device='cuda') to create tensors. (Triggered internally at /pytorch/torch/csrc/tensor/python_tensor.cpp:78.)
  output = torch.cuda.IntTensor(B, npoint)
eval: 100%|████████████████████████████████████████████████████████████████████████████████████| 472/472 [01:54<00:00,  4.11it/s, recall_0.3=(16851, 16863) / 17558]
2025-12-23 17:25:03,300   INFO  *************** Performance of EPOCH 10 *****************
2025-12-23 17:25:03,300   INFO  Generate label finished(sec_per_example: 0.0305 second).
2025-12-23 17:25:03,300   INFO  recall_roi_0.3: 0.959733
2025-12-23 17:25:03,300   INFO  recall_rcnn_0.3: 0.960417
2025-12-23 17:25:03,300   INFO  recall_roi_0.5: 0.914569
2025-12-23 17:25:03,300   INFO  recall_rcnn_0.5: 0.924821
2025-12-23 17:25:03,300   INFO  recall_roi_0.7: 0.670748
2025-12-23 17:25:03,300   INFO  recall_rcnn_0.7: 0.737442
2025-12-23 17:25:03,347   INFO  Average predicted number of objects(3769 samples): 0.000
Segmentation fault (core dumped)

    result_str, result_dict = dataset.evaluation(
        det_annos, class_names,
        eval_metric=cfg.MODEL.POST_PROCESSING.EVAL_METRIC,
        output_path=final_output_dir
    )
In this place it happens.
